tests/lib/utils/utils-test.js

```diff
@@ -126,6 +126,14 @@ describe('isConditionalExpression', () => {
   });
 });
 
+describe('isTaggedTemplateExpression', () => {
+  const node = parse('test = hbs`lorem ipsum`;').right;
+
+  it('should check if node is a tagged template expression', () => {
+    assert.ok(utils.isTaggedTemplateExpression(node));
+  });
```

Comment:
I think it's not necessary. It's really simple and I think this test will prove to be good enough. We tested other methods the same way :)